sites/platform/src/add-services/mysql/_index.md

```diff
@@ -245,6 +342,7 @@ If neither `schemas` nor `endpoints` is included, it's equivalent to the followi
                 default_schema: main
                 privileges:
                     main: admin
+{{% /snippet %}}
```

Comment:
Just realized that there's duplicate content here. **Define permissions** section should end with "You can also specify schemas for [multiple databases](link). The rest can be removed as it's identical to what's in the section directly below.